action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Generate Release Notes'
+description: 'Automatically generates release notes for a given release tag, categorized into chapters based on labels.'
+inputs:
+  repo:
```

Comment:
You don't need repo, use `const github = require('@actions/github');` and `github.context`

Reply:
Refactored.